### PR TITLE
New release template updates

### LINKED ISSUE #13262: New release template ticket updates

### PR DESCRIPTION
**Pull request by hackergil**
Fixes #13262

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-release.md [hunk 1]
```diff
@@ -92,8 +95,10 @@ Do the following steps to keep track of spartacussampledata releases:
     - [ ] `npm run release:smartedit:with-changelog` (needed since `3.2.0-next.0`)
     - [ ] `npm run release:qualtrics:with-changelog` (needed since `3.1.0-next.0`)
     - [ ] `npm run release:product-configurator:with-changelog` (needed since `3.1.0-next.0`)
-    - [ ] (for <3.2.0 releases set the spartacus peerDependencies manually, then)  
-      `npm run release:cdc:with-changelog` (since 3.2.0 release like any other lib with the same version as everything else. For older versions since 2.1.0-next.0 - publish under `0.<packages-version>.0` eg. `0.201.0-next.0` for first `2.1.0-next.0` release)
+    - [ ] `npm run release:cdc:with-changelog` (needed since `3.1.0-next.0`)
+      - [ ] For < 3.2.0 releases ONLY, set the spartacus peerDependencies manually, then)
```
Comment (Zeyber) [inline .github/ISSUE_TEMPLATE/new-release.md]: There are closing `)` on this and the next line. Where is the `(`?

**Review by Zeyber — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/new-release.md (hunk 1 above)
Comment (hackergil) [reply .github/ISSUE_TEMPLATE/new-release.md]: Deleted.

File: .github/ISSUE_TEMPLATE/new-release.md [hunk 2]
```diff
@@ -96,7 +97,7 @@ Do the following steps to keep track of spartacussampledata releases:
     - [ ] `npm run release:qualtrics:with-changelog` (needed since `3.1.0-next.0`)
     - [ ] `npm run release:product-configurator:with-changelog` (needed since `3.1.0-next.0`)
     - [ ] `npm run release:cdc:with-changelog` (needed since `3.1.0-next.0`)
-      - [ ] For < 3.2.0 releases ONLY, set the spartacus peerDependencies manually, then)  
+      - [ ] For < 3.2.0 releases ONLY, set the spartacus peerDependencies manually, then run
       `npm run release:cdc:with-changelog`.
       - [ ] For older versions since 2.1.0-next.0 ONLY, publish under `0.<packages-version>.0` eg. `0.201.0-next.0` for first `2.1.0-next.0` release)
```
Comment (Zeyber) [inline .github/ISSUE_TEMPLATE/new-release.md]: Same issue with the ending `)` without `(`.

**Review by Zeyber — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/new-release.md (hunk 2 above)
Comment (hackergil) [reply .github/ISSUE_TEMPLATE/new-release.md]: there it is!

**Pull request merged**